go/vt/vtgate/planbuilder/testdata/ddl_cases.json

```diff
@@ -39,6 +39,46 @@
       ]
     }
   },
+  {
+    "comment": "simple create procedure",
+    "query": "create procedure p1 (in a CHAR(3), out b INT) begin select c from x where d = e; end",
+    "plan": {
+      "Type": "DirectDDL",
+      "QueryType": "DDL",
+      "Original": "create procedure p1 (in a CHAR(3), out b INT) begin select c from x where d = e; end",
+      "Instructions": {
+        "OperatorType": "DDL",
+        "Keyspace": {
+          "Name": "main",
+          "Sharded": false
+        },
+        "Query": "create procedure p1 (in a CHAR(3), out b INT) begin select c from x where d = e; end;"
+      },
+      "TablesUsed": [
+        "main.p1"
+      ]
+    }
+  },
+  {
+    "comment": "simple create procedure with keyspace",
+    "query": "create procedure main_2.p1 (in a CHAR(3), out b INT) begin select c from x where d = e; end",
+    "plan": {
+      "Type": "DirectDDL",
+      "QueryType": "DDL",
```

Comment:
add test case with keyspace in the select query as well

Reply:
Done